mlrun/projects/pipelines.py

```diff
@@ -554,9 +553,7 @@ def run(
         return _PipelineRunStatus(id, cls, project=project, workflow=workflow_spec)
 
     @staticmethod
-    def wait_for_completion(
-        run_id, project=None, timeout=None, expected_statuses=None, run_object=None
-    ):
+    def wait_for_completion(run_id, project=None, timeout=None, expected_statuses=None):
```

Comment:
I see that it wasn't used at all, but r we familiar with the history of that param? can it break something removing it?

Reply:
see 2. in description